### ISSUE TITLE
Haiku port 

### ISSUE THREAD

**Issue by baimafeima** [gkarsay/parlatype#52]
As I understand parlatype is gtk-based. Unfortunately, only qt is supported on [Haiku](https://www.haiku-os.org/). Is it possible to help port it to Haiku or develop a native Haiku application as well? I believe parlatype is a best-in-class application and while I currently use it in Solus, I am planning to use Haiku more intensively in the near future.

**Comment by gkarsay** [comment]
I'm glad you like Parlatype but that's something I won't manage to do. A big, maybe the biggest part of the code is responsible for showing the wave form and that's implemented as a custom GTK+-based widget. Rewriting this in Qt is like rewriting the whole program.

I have no expertise in Qt at all. Even if somebody would step up and port it to Qt (or any other toolkit) I wouldn't accept the merge request as a single, one-time contribution. Whoever would do this, would have step up as a co-maintainer.

To summarize: I can't do this myself. The only solution is to find somebody who would like to do it and to maintain it further on, which is rather unlikely.

**Comment by baimafeima** [comment]
Yeah that makes sense. Maybe Haiku will support gtk in the future as well and this won't be a problem after all.
